### PR TITLE
Convert ERTP to Ava

### PR DESCRIPTION
**Pull request by dtribble**
refs #1568

### CODE REVIEW

**Review by warner — CHANGES_REQUESTED** [review]
there's a bogus `packages/er.tar` in the commit, as well as `packages/old/`

also bogus generated files like `testdb-lmdb/data.mdb`, `testdb/swingset-kernel-state.jsonlines`

these all need to be removed

Once those are removed (and squashed, so the final history doesn't add+remove them), and CI fixed, and the top-level unused `ava.config.js` is removed, I think this is good to land.

**Review by katelynsills — CHANGES_REQUESTED** [review]
I concur with Brian. There are some additional files that should be removed, but most of the test changes look good. There are some uses of mathHelpers that seem to be an accident, where the original was correct.

File: packages/ERTP/package.json [hunk 1]
```diff
@@ -50,15 +50,22 @@
...
   "files": [
     "src",
     "NEWS.md"
   ],
+  "ava": {
+    "files": [
+      "test/**/test-*.js"
+    ],
+    "require": [
+      "@agoric/install-ses",
```
[the remark this answers is not in the document]
Comment (dtribble) [reply packages/ERTP/package.json]: Yes that fixed it. I'll fold this into the package.json.

File: ava.config.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
+export default {
```
[the remark this answers is not in the document]
Comment (dtribble) [reply ava.config.js]: OK it looks like all the files laying around got dragged into the branch. I'll get rid of them

File: packages/ERTP/test/unitTests/test-issuerObj.js [hunk 3]
```diff
@@ -161,147 +142,139 @@ test('purse.makeDepositFacet', t => {
...
+  });
 });
 
-test('issuer.claim', t => {
+test('issuer.claim', async t => {
   t.plan(3);
   const { issuer, amountMath, mint } = makeIssuerKit('fungible');
   const payment1 = mint.mintPayment(amountMath.make(2));
-  E(issuer)
+  return E(issuer)
```
[the remark this answers is not in the document]
Comment (dtribble) [reply packages/ERTP/test/unitTests/test-issuerObj.js:179]: I agree. Moving to `await` here seems like the right thing

File: packages/ERTP/test/unitTests/test-issuerObj.js [hunk 4]
```diff
@@ -82,23 +64,22 @@ test('issuer.makeEmptyPurse', t => {
...
+    t.assert(
       amountMath.isEqual(purse.getCurrentAmount(), amountMath.getEmpty()),
       `the purse is empty again`,
     );
   };
 
-  E(purse)
+  return E(purse)
     .deposit(payment, fungible837)
     .then(checkDeposit)
```
[the remark this answers is not in the document]
Comment (dtribble) [reply packages/ERTP/test/unitTests/test-issuerObj.js:79]: I think I agree with the earlier conversation that using `await` at the end is better for overall consistency.

File: packages/ERTP/test/unitTests/test-issuerObj.js [hunk 5]
```diff
@@ -1,57 +1,39 @@
...
-      ),
-    );
-    t.equals(fungible(4000).value, 4000);
-    t.equals(fungible(0).brand, brand);
-  } catch (e) {
-    t.assert(false, e);
-  } finally {
-    t.end();
-  }
+test('issuer.getAmountMath', async t => {
```
[the remark this answers is not in the document]
Comment (dtribble) [reply packages/ERTP/test/unitTests/test-issuerObj.js:18]: definitely bad merge. The change to MathKinds came after I forked for the ava spike, and the merge of those changes wasn't right.

**Pull request closed without merging**